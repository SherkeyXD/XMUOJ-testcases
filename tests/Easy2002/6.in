108.571
138.286
38.2857
135.714
29.4286
102
42.7143
116.571
81.1429
62.8571
103.429
2.42857
30.8571
41.5714
120
53.2857
104.714
65.4286
31.4286
48.2857
20.7143
140