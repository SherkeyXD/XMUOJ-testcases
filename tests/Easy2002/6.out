5360806.476
11077028.153
235070.641
10470383.011
106757.261
4445177.741
326443.854
6635294.173
2237897.499
1040283.324
4634635.597
59.999
123070.493
300934.907
7238229.581
633753.895
4809532.510
1173252.528
130036.036
471567.874
37230.546
11494040.492
